lib/ohai/plugins/windows/cpu.rb

```diff
@@ -54,9 +54,11 @@
       cpu[current_cpu]["vendor_id"] = processor["manufacturer"]
       cpu[current_cpu]["family"] = processor["family"].to_s
       cpu[current_cpu]["model"] = processor["revision"].to_s
-      cpu[current_cpu]["stepping"] = processor["stepping"].nil? \
-                  ? processor["description"].match(/Stepping\s+(\d+)/)[1] \
-                  : processor["stepping"]
+      cpu[current_cpu]["stepping"] = if processor["stepping"].nil?
```

Comment:
you didn't see the one in chef/chef where someone used a multiline ternary and broke a string in the middle with \ in order to line wrap it and get it under 80 columns.  completely unreadable omgwtfbbq.

this example is just from someone who didn't understand whitespace parsing and tokenization in ruby...